keyboards/id80/id80.h

```diff
@@ -35,3 +35,22 @@
     { K07,   K17,   K27,   K37,   K47,   K57,   K67,   K77,   K87,   K97,   KA7   }, \
     { K08,   K18,   K28,   K38,   K48,   K58,   K68,   K78,   K88,   K98,   KC_NO }, \
 }
+
+#define LAYOUT_iso( \
+    K50,  K51, K52, K53, K54,  K55, K56, K57, K58,  KA7, KA5, KA4, KA3,   KA6,  KA2,  \
+    K40, K41, K42, K43, K44, K45, K46, K47, K48, K98, K97, K95, K94, K96,       K92,  \
+    K30,   K31, K32, K33, K34, K35, K36, K37, K38, K88, K87, K85, K84, K83,     K82,  \
+    K20,    K21, K22, K23, K24, K25, K26, K27, K28, K78, K77, K75, K74,               \
```

Comment:
QMK custom is that Enter key keycodes should be toward the end of the home row.

(1/9)

```suggestion
    K30,   K31, K32, K33, K34, K35, K36, K37, K38, K88, K87, K85, K84,          K82,  \
    K20,    K21, K22, K23, K24, K25, K26, K27, K28, K78, K77, K75, K74, K83,          \
```